test/sparse/sparse.jl

```diff
@@ -661,6 +661,53 @@ end
         end
     end
 
+    @testset "spones" begin
```

Comment:
this still isn't an "issue" - by original location I meant where they were in the file before your PR, not at an earlier commit